project/AkkaDisciplinePlugin.scala

```diff
@@ -64,10 +64,14 @@ object AkkaDisciplinePlugin extends AutoPlugin {
     "akka-testkit")
 
   lazy val silencerSettings = {
-    val silencerVersion = "1.7.1"
+    val silencerVersion = "1.5-SNAPSHOT"
+    // TODO: FIXME DOTTY (using a locally published version of silencer...)
```

Comment:
I simply published locally an "empty version" of the silencer, any plan to port it to Scala 3 already on-going @ghik ?

Reply:
Does Scala 3 support `@nowarn`? I would expect that it does. All `@silent` annotations should be migrated to `@nowarn` - silencer would then be necessary only in Scala 2.12 (it understands `@nowarn` to some extent).

Reply:
Thanks @ghik for the answer!
The question then goes to the Akka Team ( @johanandren @raboof ): can we migrate all the annotations to `@nowarn` in the entire code-base, and tweak compiler flags to be more lenient in Scala 2.12?
Or, we can define a local annotation `@nowarn` in the `scala-212` folder that simply rebind `silence` (have to test this option).

Reply:
`@nowarn` is already available for Scala 2.12 in [scala-collection-compat](https://github.com/scala/scala-collection-compat/blob/181d8128a4505e1d592d5d298aeb1a26da8ed12b/compat/src/main/scala-2.11_2.12/scala/annotation/nowarn.scala). It is [partially interpreted](https://github.com/ghik/silencer#using-nowarn) by silencer.

Reply:
`@nowarn` will be in Scala 2.12.13 (https://github.com/scala/scala/pull/9248), so I think it'd make sense to wait for that.

Do we need to make our flags more lenient in order to move to `@nowarn`?

Reply:
> Do we need to make our flags more lenient in order to move to @nowarn?

Don't know, will test!

Reply:
`@nowarn` is not implemented in Scala 3 yet: https://github.com/lampepfl/dotty/issues/8908